src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/percent_widget.html.php

```diff
@@ -1 +1,13 @@
-<?php echo $view['form']->block($form, 'form_widget_simple', ['type' => isset($type) ? $type : 'text']) ?> %
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$symbol = $symbol !== false ? (!empty($symbol) ? ' ' . $symbol : ' %') : '';
```

Comment:
no whitespace around `.`

Reply:
could be: `$symbol = false !== $symbol ? ($symbol ? ' '.$symbol : ' %') : '';`

Reply:
Ok 👌🏻